documentation/operators/src/nodes/gateway-setup.md

```diff
@@ -56,39 +60,110 @@ To check available configuration options use:
+<!-- cmdrun ../../../../target/release/nym-gateway init --id supergateway --host $(curl ifconfig.me) -->
 ```
 ~~~
 
 The `$(curl ifconfig.me)` command above returns your IP automatically using an external service. Alternatively, you can enter your IP manually if you wish. If you do this, remember to enter your IP **without** any port information.
 
+
+#### Initialising gateway with network requester
+
+As some of the [Project Smoosh](../faq/smoosh-faq.md) changes getting implemented, the gateways now can work also as a network requesters at the same time. Such combination creates an exit gateway node, needed for new more open setup. 
+
+An operator can initialise the gateway and network requester together by running:
+
+```
+./nym-gateway init --id <ID> --host $(curl ifconfig.me) --with-network-requester
+```
+
+If we follow the previous example with `<ID>` chosen `superexitgateway`, adding the `--with-network-requester` flag, the outcome will be:
+
+
+~~~admonish example collapsible=true title="Console output"
+```
+<!-- cmdrun ../../../../target/release/nym-gateway init --id superexitgateway --host $(curl ifconfig.me) --with-network-requester -->
+```
+~~~
+
+You can see that the printed information besides identity and sphinx keys also includes a long string called address. This is the address to be provided to your local [socks5 client](https://nymtech.net/docs/clients/socks5-client.html) as a `--provider` if you wish to connect to your own exit gateway.  
+
+#### Add network requester to existing gateway
+
+If you already have a gateway and got it [upgraded](./maintenance.md#upgrading-your-node) to the [newest version](./gateway-setup.md#current-version), you can easily add a network requester by stopping your gateway and running a command `setup-network-requester`.
+
+See the options:
+
+```
+./nym-gateway setup-network-requester --help
+```
+
+~~~admonish example collapsible=true title="Console output"
+```
+<!-- cmdrun ../../../../target/release/nym-gateway setup-network-requester --help -->
+```
+~~~
+
+Run with `--enabled true` flag choosing `<ID>` as `supergateway`:
+
+```
+./nym-gateway setup-network-requester --enabled true --id supergateway 
+```
+
+~~~admonish example collapsible=true title="Console output"
+```
+<!-- cmdrun ../../../../target/release/nym-gateway setup-network-requester --enabled true --id supergateway -->
+```
+~~~
+
+In case there are any problems, you can also change it manually by editing the gateway config stored in `/home/user/.nym/gateways/<ID>/config/config.toml` where the line under `[network_requester]` needs to be edited from `false` to `true`, it shall look like this:  
+
+```
+[network_requester]
+# Specifies whether network requester service is enabled in this process.
+enabled = true
+```
+
+Save, exit and restart your gateway.
+
+All information about your network requester connected to your gateway is in `/home/user/.nym/gateways/snus/config/network_requester_config.toml`.
```

Comment:
see above re naming

Reply:
i go with exit gateway everywhere to entrench it.